### ISSUE TITLE
Poor accessibility for blind users

### ISSUE THREAD

**Issue by cyrmax** [transmitic/transmitic#120]
## The problem

Blind users can interact with computers using screen reading software (screenreaders) such as NVDA, JAWS, Windows Narrator, ETC.
Transmitic has very poor accessibility support for blind users.

I'm totally blind and can help with testing, if accessibility issues are fixed.

## Short universal checklist

* Every UI element is focusable with keyboard with tab and shift+tab (see below for special cases);
* Lists and radio button groups are focusable with Tab and shift+tab and focus drops on the selected item inside;
* Navigation between list items or radio buttons is performed with vertical arrows on keyboard;
* Checkboxes are toggleable with space key;
* Buttons are pressable either with space or with enter key (better if both);
* Every UI element has descriptive accessibility label (for example "Connected users, list, user 1 not selected 1 of 5" or "use reverse connection, checkbox not checked");
* If there is some help or information text inside a window it should be wrapped in a read-only text field for screenreaders to be able to focus on it and allow blind users to read the text word by word, copy it or even translate;
* All described above should be achieved with native methods (such as UIA or MSAA on Windows, ATSPI on Linux and UIAccessibility on MacOS), no need to create as called "self-voicing UI" as it is more difficult to implement and support in future.

**Comment by Andrew-Shay** [comment]
Thanks for the feedback. The UI uses https://sciter.com/. I'll look into the accessibility features and see what I need to change.
